contracts/storage/app.sol

```diff
@@ -26,6 +26,7 @@ struct AppStorage {
     TellerNFT nft;
     UpgradeableBeaconFactory loansEscrowBeacon;
     UpgradeableBeaconFactory collateralEscrowBeacon;
+    address nftLiquidator;
```

Comment:
rename to `nftLiquidationController`

we can eventually make this a contract that implements the `IERC721Receiver(address).onERC721Received(...)` function to auto list on opensea